src/components/select/select.js

```diff
@@ -782,38 +785,40 @@ function SelectProvider($$interimElementProvider) {
      * Interim-element onRemove logic....
      */
     function onRemove(scope, element, opts) {
+      opts = opts || { };
```

Comment:
If the user fails to pass options (or passes null), won't the next three lines fail because we've set it to an empty object which does not have those methods?